frontend/src/routes/manage/Realm/Content/Edit/EditMode/Series.tsx

```diff
@@ -36,7 +36,7 @@ export const EditSeriesBlock: React.FC<EditSeriesBlockProps> = ({ block: blockRe
 
     const { series: allSeries } = useFragment(graphql`
         fragment SeriesEditModeSeriesData on Query {
-            series { id title }
+            series: allSeries { id title }
```

Comment:
This is now a bit roundabout. The variable was called `allSeries` already exactly because of the problem you mention in the commit message. Now you rename the field to `series` in the query only to destructure-rename it as `allSeries` in the assignment ...

Reply:
Makes sense :smile: